### DISCUSSION TITLE
How do you filter by text in url?

### DISCUSSION THREAD

**Discussion by officiallymarky** [tweepy/tweepy#1385] [Questions] [answered]
I tried track=['filter:links url:reddit']) and track=['url:reddit']) neither which seem to work for streaming

**Comment by Harmon758** [comment, marked as answer]
Unfortunately, this is a limitation with Twitter's API:
> You cannot use the search operators with statuses/filter.

https://twittercommunity.com/t/statuses-filter-and-search-operators/115840/2
